roles/openshift_excluder/tasks/status.yml

```diff
@@ -20,13 +20,15 @@
 - name: Update to latest excluder packages
   package:
     name: "{{ openshift.common.service_type }}-excluder"
+    state: latest
```

Comment:
@sdodson @ingvagabund Is the openshift_excluder role applied outside of the upgrade playbooks? My concern with updating the excluder packages in `status.yml` is that we would update the package outside of the upgrade workflow.